tests/entities/test_run.py

```diff
@@ -53,14 +53,15 @@ def test_string_repr(self):
             source_name="source-name", entry_point_name="entry-point-name",
             user_id="user-id", status=RunStatus.FAILED, start_time=0, end_time=1,
             source_version="version", lifecycle_stage=LifecycleStage.ACTIVE)
-        metrics = [Metric("key", i, 0) for i in range(5)]
+        metrics = {"key-%s" % i: Metric("key-%s" % i, i, 0) for i in range(3)}
         run_data = RunData(metrics=metrics, params=[], tags=[])
         run1 = Run(run_info, run_data)
-        expected = "<Run: data=<RunData: metrics=[<Metric: key='key', timestamp=0, value=0>, " \
-                   "<Metric: key='key', timestamp=0, value=1>, ...], params=[], tags=[]>, " \
-                   "info=<RunInfo: artifact_uri=None, end_time=1, " \
-                   "entry_point_name='entry-point-name', experiment_id=0, " \
-                   "lifecycle_stage='active', name='name', run_uuid='hi', " \
-                   "source_name='source-name', source_type=3, source_version='version', " \
-                   "start_time=0, status=4, user_id='user-id'>>"
+        expected = ("<Run: data=<RunData: metrics={'key-0': <Metric: key='key-0', timestamp=0, "
+                    "value=0>,\n 'key-1': <Metric: key='key-1', timestamp=0, value=1>,\n "
```

Comment:
It looks like Python's pprint library adds these newlines between successive key-value pairs in the same dictionary, which I guess improves readability a bit.